### DISCUSSION TITLE
How to run multiple prompts?

### DISCUSSION THREAD

**Discussion by Mr-Jeffery** [ggml-org/llama.cpp#4855] [Q&A]
I am running llamav2, I have a file with multiple prompts in seperate lines, but the model seems to intepret them as one single prompt, how should I prepare my prompts?

**Comment by Alex4210987** [comment]
same question lol

**Reply by Mr-Jeffery** [reply]
I solved my problem by rewriting `simple.cpp` myself, I changed my prompts into a json and let the script to read from it. My script is attached here:
[modified_simple.zip](https://github.com/user-attachments/files/16744918/modified_simple.zip)
For simplicity, you can just rename the file back to `simple.cpp` and type make. Let me know if you have further question

**Reply by Alex4210987** [reply]
thx, i will try later
